src/main/java/io/neonbee/NeonBee.java

```diff
@@ -592,6 +593,15 @@ public void unregisterLocalConsumer(String verticleAddress) {
         localConsumers.remove(verticleAddress);
     }
 
+    /**
+     * Returns the ServerConfig if NeonBee is started with WEB profile.
+     *
+     * @return the ServerConfig or null if ServerVerticle is not started.
+     */
+    public ServerConfig getServerConfig() {
+        return new ServerConfig((JsonObject) getLocalMap().get(ServerVerticle.SERVER_CONFIG_KEY));
```

Comment:
Instead of storing the JSON object, how about storing the ServerConfig directly, that way it will not need to get parsed everytime again.